dll/win32/wdmaud.drv/wdmaud.h

```diff
@@ -144,6 +144,18 @@ WdmAudGetWavePositionByMMixer(
     IN  struct _SOUND_DEVICE_INSTANCE* SoundDeviceInstance,
     IN  MMTIME* Time);
 
+MMRESULT
+WdmAudGetVolumeByMMixer(
+    _In_ struct _SOUND_DEVICE_INSTANCE* SoundDeviceInstance,
```

Comment:
Please use `PSOUND_DEVICE_INSTANCE` instead of `struct _SOUND_DEVICE_INSTANCE*` in your new code
(except in the mmebuddy header, for the time being, since it appears the typedef that allows to use the shorter name is defined after the functions...)

Reply:
Done. Changed it in this file and in wdmaud.h header.